requirements.txt

```diff
@@ -1,5 +1,5 @@
 asgiref==3.5.0
-backports.zoneinfo==0.2.1
+backports.zoneinfo;python_version<"3.9"
```

Comment:
what's this?

Reply:
we needed to add this in order to setup the server on joyce's laptop

Reply:
Added this to get my development environment working